affiliated/registry.json

```
@@ -1,5 +1,23 @@
 {
     "packages": [
+        {
+          "name": "PyAutoLens",
+          "maintainer": "James Nightingale <james.w.nightingale@durham.ac.uk>",
+          "stable": true,
+          "home_url": "https://pyautolens.readthedocs.io/en/latest/",
+          "repo_url": "https://github.com/Jammy2211/PyAutoLens",
+          "pypi_name": "https://pypi.org/project/autolens/",
+          "description": "Analysis, modeling and simulations of strong gravitational lenses.",
```

```suggestion
          "pypi_name": "autolens",
          "description": "Analysis, modeling and simulations of strong gravitational lenses.",
          "coordinated": false,
```
Only use "project" name on PyPI; + required field.